data_import/serializers.py

```diff
@@ -43,3 +44,46 @@ def to_representation(self, instance):
         ret["source"] = instance.source
 
         return ret
+
+
+class NewDataFileAccessLogSerializer(serializers.ModelSerializer):
+    """
+    Serialize the logs of file access events from Open Human's end.
```

Comment:
`Serialize logs of file access requests for custom API endpoint for OHLOG_PROJECT_ID`